common/djangoapps/student/migrations/0047_add_has_profile_image_boolean.py

```diff
@@ -0,0 +1,194 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
```

Comment:
It looks like whichever one of us lands second will have to re-generate the migrations.